src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -104,6 +104,21 @@ def translate_facets(cls, facets: JSON, translation: JSON):
                 bucket['key'] = Document.translate_field(bucket['key'], path, forward=False)
         return facets
 
+    @classmethod
+    def filter_contains_a_null(cls, field: str, filter_value: list) -> bool:
+        """
+        Return true if filter_value contains a translated None/null value
+        :param field: A field in Elasticsearch (eg. 'contents.samples.effective_organ')
+        :param filter_value: The value(s) being filtered for (eg. ['__null__', 'brain'])
+        :return: If a translated null value was found in the filter_value
+        """
+        path = tuple(field.split('.'))
+        # Translate a None value for this field to get the correct value to check for based on the field type
+        translated_null = Document.translate_field(None, path=path)
+        for value in filter_value:
+            if value == translated_null:
+                return True
+        return False
```

Comment:
```suggestion
        return Document.translate_field(None, path=tuple(field.split('.'))) in filter_value
```

… and then it becomes questionable if a separate method is needed.